src/shared/utils/mixpanel.ts

```diff
@@ -1,29 +1,28 @@
 import mixpanel, { Dict } from 'mixpanel-browser';
-import UAParser from 'ua-parser-js';
-
-mixpanel.init('YOUR_MIXPANEL_TOKEN');
 
 const isProduction = process.env.REACT_APP_ENV === 'PRODUCTION';
 const isStaging = process.env.REACT_APP_ENV === 'STAGE';
 const shouldEnableMixpanel = isProduction || isStaging;
 
-const uaParser = new UAParser();
+// A project's token is not a secret value.
+// In front-end implementation this token will be available to anyone visiting the website.
+// More on this topic: https://developer.mixpanel.com/reference/project-token;
+const PROJECT_TOKEN = '075d1512e69a60bfcd9f7352b21cc4a2';
```

Comment:
Why not  this value cannot be provided via env environment? otherwise anyone can change this value in repo accidentally or intentionally. Thus the tracked data will be missed.